scripts/spotify.coffee

```diff
@@ -4,7 +4,7 @@
 #
 
 module.exports = (robot) ->
-  robot.hear spotify.link, (msg) ->
+  robot.respond spotify.link, (msg) ->
```

Comment:
This could probably stay, but I think Slack already has something like this built-in. If that's right, we could probably just kill this one altogether.